src/index.html

```diff
@@ -7,8 +7,66 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          class="logo"
+          alt="Moyo"
+          >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#Smartfones" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#Laptops & computers"
+              class="nav__link"
+              data-qa="hover"
+            >
```

Comment:
Fix the code style and don't use spaces in link 
```suggestion
            <a 
              href="#Laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
```